example/client/main.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2023 CloudWeGo Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"context"
+	"log"
+	"time"
+
+	"github.com/cloudwego/kitex-examples/kitex_gen/api"
+	"github.com/cloudwego/kitex-examples/kitex_gen/api/echo"
+	"github.com/cloudwego/kitex/client"
+	"github.com/cloudwego/kitex/pkg/klog"
+	etcdclient "github.com/kitex-contrib/config-etcd/client"
+	"github.com/kitex-contrib/config-etcd/etcd"
+	"github.com/kitex-contrib/config-etcd/utils"
+)
+
+type configLog struct{}
+
+func (cl *configLog) Apply(opt *utils.Options) {
+	fn := func(k *etcd.Key) {
+		klog.Infof("etcd config %v", k)
+	}
+	opt.EtcdCustomFunctions = append(opt.EtcdCustomFunctions, fn)
+}
+
+func main() {
+	klog.SetLevel(klog.LevelDebug)
+
+	etcdClient, err := etcd.New(etcd.Options{})
```

Comment:
Please rename the function to `etcd.NewClient(..)`

Reply:
Done